MovieRate/src/assets/rate.json

```diff
@@ -0,0 +1,32 @@
+[
```

Comment:
this array can be generated in a more compact way. And you can store it not as json but just const in regular js file.
```
const arr = new Array(10).fill(true).map((item, i) => ({value: ++i}));
```